plugins/inventory/lxd.py

```diff
@@ -0,0 +1,1108 @@
+            required: true
+            choices: [ 'community.general.lxd' ]
+        url:
+            description:
+            - The unix domain socket path or the https URL for the lxd server.
+            - Sockets in filesystem have to start with 'unix:'.
+            - Mostly C(unix:/var/lib/lxd/unix.socket) or C(unix:/var/snap/lxd/common/lxd/unix.socket).
+            default: unix:/var/snap/lxd/common/lxd/unix.socket
+            required: false
+            type: str
+        client_key:
+            description:
+            - The client certificate key file path.
+            required: false
+            aliases: [ key_file ]
+            default: $HOME/.config/lxc/client.key
+            type: path
+        client_cert:
+            description:
+            - The client certificate file path.
+            required: false
+            aliases: [ cert_file ]
+            default: $HOME/.config/lxc/client.crt
+            type: path
+        trust_password:
+            description:
+            - The client trusted password.
+            - You need to set this password on the lxd server before
+                running this module using the following command
+                'lxc config set core.trust_password <some random password>'
+                See U(https://www.stgraber.org/2016/04/18/lxd-api-direct-interaction/).
+            - If trust_password is set, this module send a request for authentication before sending any requests.
+            required: false
+            type: str
+        state:
+            description: Filter the container according to the current status.
+            required: false
+            type: str
+            default: none
+            choices: [ 'STOPPED', 'STARTING', 'RUNNING', 'none' ]
+        prefered_container_network_interface:
+            description:
+            - If a container has multiple network interfaces, select which one is the prefered as pattern.
+            - Combined with the first number that can be found e.g. 'eth' + 0.
+            required: false
+            type: str
+            default: eth
+        prefered_container_network_family:
+            description:
+            - If a container has multiple network interfaces, which one is the prefered by family e.g.
+                - inet is ipv4 or
+                - inet6 is ipv6.
+            required: false
+            type: str
+            default: inet
+            choices: [ 'inet', 'inet6' ]
+        groups:
+            description:
+            - Create Groups by
+                - 'location' e.g. grabed sample by web, maybe it works
+                - 'pattern' e.g. samba* or regex
+                - 'network_range' e.g. 192.168.0.0/24
+                - 'os' e.g. ubuntu
+                - 'release' e.g. groovy
+                - 'profile' e.g. default
+                - 'vlanid' works on defined networks e.g. br0 with vlanid_default == 666
```

Comment:
Using lists in the documentation does not work. You have to rewrite this using paragraphs (top-level list items) in some way.